noir-projects/noir-contracts/contracts/token_bridge_contract/src/main.nr

```diff
@@ -84,39 +84,42 @@ contract TokenBridge {
         context.message_portal(storage.portal_address.read_public(), content);
 
         // Burn tokens
-        Token::at(storage.token.read()).burn_public(context.msg_sender(), amount, nonce).call(
-            &mut context,
-        );
+        Token::at(storage.token.read_public())
+            .burn_public(context.msg_sender(), amount, nonce)
+            .call(&mut context);
     }
     // docs:end:exit_to_l1_public
+
     // docs:start:claim_private
-    // Consumes a L1->L2 message and calls the token contract to mint the appropriate amount in private assets
-    // User needs to call token.redeem_shield() to get the private assets
-    // TODO(#8416): Consider creating a truly private claim flow.
+    /// Claims the bridged tokens and makes them accessible in private. Note that recipient's address is not revealed
+    /// but the amount is. Hence it's most likely possible to determine to which L1 deposit this claim corresponds to
+    /// (unless there are multiple pending deposits of the same amount).
+    /// TODO(#8416): Consider creating a truly private claim flow.
     #[private]
     fn claim_private(
-        secret_hash_for_redeeming_minted_notes: Field, // secret hash used to redeem minted notes at a later time. This enables anyone to call this function and mint tokens to a user on their behalf
+        recipient: AztecAddress, // recipient of the bridged tokens
```

Comment:
We are no longer redeeming "to" a TransparentNote and instead we just mint a real ValueNote to a recipient.